src/components/AddedCourses/AddedCoursePane.js

```diff
@@ -173,7 +181,7 @@ class AddedCoursePane extends PureComponent {
                         </Grid>
                     );
                 })}
-                <Typography variant="h6">Custom Events</Typography>
+                <custEventHeader hasEvents={this.state.customEvents.length !== 0} />
```

Comment:
Good idea using `.length`! We can actually simplify this down to one line using a [conditional render](https://beta.reactjs.org/learn/conditional-rendering#logical-and-operator-). Here the Typography will only be displayed if the previous statement is true.
```suggestion
                {this.state.customEvents.length > 0 && <Typography variant="h6">Custom Events</Typography>}
```